### PR TITLE
Add correct zoom-scroll handling

### PR DESCRIPTION
**Pull request by f3dm76**

### CODE REVIEW

File: Source/views/MacawView.swift [hunk 1]
```diff
@@ -11,31 +11,178 @@ import AppKit
...
+        initializeView()
+    }
+
+    func initializeView() {
+
+        if !self.subviews.contains(drawingView) {
+            self.backgroundColor = .white
+            self.clipsToBounds = true
+            self.addSubview(drawingView)
+            drawingView.backgroundColor = .white
```
Comment (ystrot) [inline Source/views/MacawView.swift:116]: I think drawingView should be transparent. If you keep it white, you will be unable to change its color.

File: Source/animation/types/animation_generators/Cache/AnimationCache.swift [hunk 2]
```diff
@@ -8,7 +8,7 @@ import AppKit
 
 class AnimationUtils {
 
-    class func layerForNodeRenderer(_ renderer: NodeRenderer, _ context: AnimationContext, animation: Animation, customBounds: Rect? = .none, shouldRenderContent: Bool = true) -> ShapeLayer {
+    class func layerForNodeRenderer(_ renderer: NodeRenderer, animation: Animation, customBounds: Rect? = .none, shouldRenderContent: Bool = true) -> ShapeLayer {
```
Comment (amarunko) [inline Source/animation/types/animation_generators/Cache/AnimationCache.swift:11]: Maybe we should try to use maximum line length equal to 120 symbols (for example)? Long lines are really uncomfortable to read and understand, especially here on Github.

Comment (ystrot) [reply Source/animation/types/animation_generators/Cache/AnimationCache.swift:11]: Would be great to add it to our [SwiftLint](https://github.com/exyte/Macaw/blob/master/.swiftlint.yml) 😉

Comment (amarunko) [reply Source/animation/types/animation_generators/Cache/AnimationCache.swift:11]: If you agree - it takes about 5 mines to update rules for SwiftLint and maybe some time to format current issues.

**Pull request merged**